Python/Product/Profiling/getArch.py

```diff
@@ -0,0 +1,3 @@
+﻿import struct
+
+print(struct.calcsize("P")*8)
```

Comment:
@zooba I had Stella move ahead with this just so she wouldn't be blocked, but I know this isn't exactly what you wanted. I figured we could fix this up in the PR, so here we are. Can you please remind us how you'd prefer this to be done here? Thanks!

Reply:
Check for `sys.winver.endswith('-32')` or `sys.winver.endswith('-arm64')`.

Reply:
And it looks like amd64 will just return the major.minor. I ran `sys.winver` on my 64-bit python install and I got `3.9`. So those cases should be enough to determine the arch of the python exe